plugins/suspendmanager.cpp

```diff
@@ -957,10 +963,10 @@ static bool suspendmanager_isKeptSuspended(df::job *job) {
     }
 }
 
-static void suspendmanager_runOnce(color_ostream &out, bool blocking) {
+static void suspendmanager_runOnce(color_ostream &out, bool blocking, bool unsuspend_everything) {
     auto save = suspendmanager_instance->prevent_blocking;
     suspendmanager_instance->prevent_blocking = blocking;
-    do_cycle(out);
+    suspendmanager_instance->do_cycle(out, unsuspend_everything);
```

Comment:
the pattern common to other `do_cycle` tools is to update the last run tick counter when manually run like this, but I agree that it's not a big deal to skip the update for a tool that runs this frequently when enabled.